var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -15,9 +15,7 @@ class Relion(CMakePackage, CudaPackage):
     homepage = "http://http://www2.mrc-lmb.cam.ac.uk/relion"
     git      = "https://github.com/3dem/relion.git"
 
-    version('3.0.6', preferred='true', tag='3.0.6')
-    version('2.1', tag='2.1')
-    version('2.0.3', tag='2.0.3')
+    version('3.0.7', preferred='true', tag='3.0.7')
```

Comment:
No need for `preferred=True` (I'm not even sure how it works when it isn't a Boolean). Spack automatically chooses the latest stable release.

Reply:
Fixed, thank you!